source/docs/software/can-devices/can-addressing.rst

```diff
@@ -185,3 +185,18 @@ For CAN Nodes to be accepted for use in the FRC System, they must:
 -  Support the minimum Broadcast message requirements as detailed in the Broadcast Messages section.
 -  If controlling actuators, utilize a scheme to assure that the robot is issuing commands, is enabled, and is still present
 -  Provide software library support for LabVIEW, C++, and Java or arrange with *FIRST*\ |reg| or FIRSTs Control System Partners to provide such interfaces.
+
+Universal Heartbeat
+-------------------
+
+Starting in 2022, the roboRIO provides a universal CAN heartbeat. This heartbeat is sent every 20ms.
+The heartbeat has a full CAN ID of 0x01011840, which it NI manufacturer, RobotController type, Device ID 0 and API ID 0x062.
```

Comment:
```suggestion
The heartbeat contains: a full CAN ID of 0x01011840 (which is the NI manufacturer id), the RobotController type, Device ID 0 and API ID 0x062.
```

Reply:
Contains isn't the right word here. The Id is a descriptor and not contents.

Reply:
I fixed the wording up some.